src/sentry/static/sentry/app/views/alerts/details/index.tsx

```diff
@@ -154,4 +153,4 @@ class IncidentDetails extends React.Component<Props, State> {
   }
 }
 
-export default withApi(withOrganization(IncidentDetails));
+export default withApi(IncidentDetails);
```

Comment:
removed this as the `AlertsContainer` now populates its children (`IncidentDetails`, `AlertRulesListContainer`) with an organization